commit ad61df918c44316940404891d5082c63e79c256a
Author: Jiri Pirko <jpirko@redhat.com>
Date:   Thu Oct 8 01:21:46 2009 -0700

    netlink: fix typo in initialization
    
    Commit 9ef1d4c7c7aca1cd436612b6ca785b726ffb8ed8 ("[NETLINK]: Missing
    initializations in dumped data") introduced a typo in
    initialization. This patch fixes this.
    
    Signed-off-by: Jiri Pirko <jpirko@redhat.com>
    Signed-off-by: David S. Miller <davem@davemloft.net>

Adjusted to apply to Debian's 2.6.26 by dann frazier <dannf@debian.org>

diff -urpN linux-source-2.6.26.orig/net/sched/cls_api.c linux-source-2.6.26/net/sched/cls_api.c
--- linux-source-2.6.26.orig/net/sched/cls_api.c	2008-07-13 15:51:29.000000000 -0600
+++ linux-source-2.6.26/net/sched/cls_api.c	2009-10-27 21:47:00.000000000 -0600
@@ -333,7 +333,7 @@ static int tcf_fill_node(struct sk_buff 
 	tcm = NLMSG_DATA(nlh);
 	tcm->tcm_family = AF_UNSPEC;
 	tcm->tcm__pad1 = 0;
-	tcm->tcm__pad1 = 0;
+	tcm->tcm__pad2 = 0;
 	tcm->tcm_ifindex = tp->q->dev->ifindex;
 	tcm->tcm_parent = tp->classid;
 	tcm->tcm_info = TC_H_MAKE(tp->prio, tp->protocol);
